sdk/ml/azure-ai-ml/azure/ai/ml/_internal/entities/_additional_includes.py

```diff
@@ -68,7 +68,18 @@ def _copy(src: Path, dst: Path) -> None:
         if src.is_file():
             _general_copy(src, dst)
         else:
-            shutil.copytree(src, dst)
+            shutil.copytree(src, dst, ignore=shutil.ignore_patterns("__pycache__"))
+
+    @staticmethod
+    def _zip_need_to_compress(path: Path) -> bool:
```

Comment:
add some docstring for this func

Reply:
and maybe rename to something like _is_folder_to_compress? Indicate this is a function to check something

Reply:
added docstring and renamed this